Resources/doc/index.rst

```diff
@@ -0,0 +1,44 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+```bash
+$ composer require symfony/swiftmailer-bundle "~2.3"
+```
```

Comment:
this is markdown syntax, but the docs have to be written in rst in order to be rendered on symfony.com. This means you have to change this to:

```rst
.. code-block:: bash

    $ composer require symfony/swiftmailer-bundle "~2.3"
```